libs/sr/page-team/src/index.tsx

```diff
@@ -105,51 +121,112 @@ function Page({ teamId }: { teamId: string }) {
       teammateDatum,
     }
   }, [teammateDatum, team, teamId])
-
+  const srcDstDisplayContextValue = useMemo(() => {
+    const charDisplay = objKeyMap(
+      moveToFront(
+        team.teamMetadata
+          .filter(notEmpty)
+          .map(({ characterKey }) => characterKey),
+        characterKey as CharacterKey
+      ),
+      (ck) => <CharacterName characterKey={ck} />
+    )
+    return { srcDisplay: charDisplay, dstDisplay: charDisplay }
+  }, [team.teamMetadata, characterKey])
+  const conditionals = useMemo(
+    () =>
+      team.conditionals.map(({ sheet, src, dst, condKey, condValues }) => ({
+        sheet,
+        src,
+        dst,
+        condKey,
+        condValue: condValues[presetIndex],
```

Comment:
UI is rendering conditionals based on DB value, not engine value. not sure if that is intended

Reply:
Don't think it matters too much right now, since engine does not validate or change the value.
Rendering DB is faster than waiting for a calc refresh. 
I guess in terms of debug having the calc value is more valuable.

Reply:
Won't engine validate list and num conditionals to be one of the defined values or within the min/max respectively? Or is that only handled within the sheet itself

Reply:
I believe meta.ts provides the parameter for UI/DB to validate the value, but engine does not use them.